javatests/arcs/showcase/typevariables/Consumer.kt

```diff
@@ -0,0 +1,25 @@
+package arcs.showcase.typevariables
+
+import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.Job
+
+class Consumer : AbstractConsumer() {
+
+    /** Process an order. In this case, we ensure skus have been redacted and prices are fair. */
+    override fun onUpdate() =
+        handles.data.fetchAll()
+            .also { it.size == 3 }
```

Comment:
`.also { it.size == 3 }` does not do anything. Am I missing something here?

Reply:
I just noticed this this morning -- thanks for catching, it should be an assert.